bulbs/api/views.py

```diff
@@ -75,7 +81,7 @@ class ContentViewSet(UncachedResponse, viewsets.ModelViewSet):
         "authors", "types"
     )
     permission_classes = [IsAdminUser, CanEditContent]
-    metadata_class = ContentViewMetaData
+    metadata_class = InfographicMetadataMixin
```

Comment:
Ah. this is something we should change, but it's a product of our polymorphic ContentView hierarchy.